Assets/HoloToolkit-Examples/Input/Scripts/InputTest.cs

```diff
@@ -9,16 +9,9 @@ namespace HoloToolkit.Unity.InputModule.Tests
     /// Test behaviour that simply prints out a message very time a supported event is received from the input module.
     /// This is used to make sure that the input module routes events appropriately to game objects.
     /// </summary>
-    public class InputTest : MonoBehaviour, 
-                             IInputHandler,
-                             IInputClickHandler,
-                             IFocusable, 
-                             ISourceStateHandler,
-                             IHoldHandler,
-                             IManipulationHandler,
-                             INavigationHandler
+    public class InputTest : MonoBehaviour, IInputHandler, IInputClickHandler, IFocusable, ISourceStateHandler, IHoldHandler, IManipulationHandler, INavigationHandler
     {
```

Comment:
Should we be including the IPointerSpecificFocusable in here?  Or do we want that as a separate issue?